packages/browser/core/src/core/auth/internal/StitchAuthImpl.ts

```diff
@@ -15,39 +15,72 @@
  */
 
 import {
+  AuthInfo,
   CoreStitchAuth,
   CoreStitchUser,
   DeviceFields,
   StitchAppClientInfo,
-  StitchAuthRoutes,
+  StitchAuthResponseCredential,
   StitchCredential,
+  StitchError,
   StitchRequestClient,
   StitchUserFactory,
-  Storage
+  Storage,
 } from "mongodb-stitch-core-sdk";
 
 import { detect } from "detect-browser";
+import version from "../../internal/common/Version";
 import AuthProviderClientFactory from "../providers/internal/AuthProviderClientFactory";
 import NamedAuthProviderClientFactory from "../providers/internal/NamedAuthProviderClientFactory";
+import StitchRedirectCredential from "../providers/StitchRedirectCredential";
 import StitchAuth from "../StitchAuth";
 import StitchAuthListener from "../StitchAuthListener";
 import StitchUser from "../StitchUser";
+import StitchBrowserAppAuthRoutes from "./StitchBrowserAppAuthRoutes";
 import StitchUserFactoryImpl from "./StitchUserFactoryImpl";
-import version from "../../internal/common/Version";
 
-export default class StitchAuthImpl extends CoreStitchAuth<StitchUser>
+export enum RedirectKeys {
+  RedirectProvider = "redirectProvider",
+  Type = "_stitch_redirect_type",
+}
+
+export enum RedirectFragmentFields {
+  StitchError = "_stitch_error",
+  State = "_stitch_state",
+  UserAuth = "_stitch_ua",
+  LinkUser = "_stitch_link_user",
+  StitchLink = "_stitch_link",
+}
+
+export enum RedirectTypes {
+  Link = "isLink",
```

Comment:
[nit] why not just name them link and login?

Reply:
👍